### PR TITLE
OpenContext setup, add SESAR CV mapping

### PR DESCRIPTION
**Pull request by hyunssong**
This is a PR that covers some old commits I missed to push previously.
Rebased the branch on the updated `develop` .

### CODE REVIEW

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -31,28 +31,36 @@
...
      - name: flake8 linter
        run:  |
          source ./.venv/bin/activate
          python -m flake8 --count --max-complexity 10 --ignore E501,W503 --show-source --statistics --exclude ./.venv .
      - name: Run mypy
        run: |
-         source ./.venv/bin/activate       
+         source ./.venv/bin/activate
          pip install mypy
          mypy --install-types --non-interactive --namespace-packages
```
Comment (dannymandel) [inline .github/workflows/python-app.yml:66]: Something is fishy here -- these are my changes from a different pull request that has merged.  It leads me to believe this PR is missing changes from current `develop`

**Comment by dannymandel** [comment]
This looks good to go provided we get latest develop merged in -- there shouldn't be any changes in `python-app.yml` in this PR.  merge or rebase should fix it.  Thanks!

**Comment by dannymandel** [comment]
I'll approve so this doesn't get stuck waiting, but please make sure you rebase the changes on latest develop.  It looks like the history on this branch is very old, so you might consider creating a new branch/PR and cherry-picking the changes over there if you get stuck.

**Review by dannymandel — APPROVED** [review]
Looks good, please make sure to merge/rebase latest develop.

File: .github/workflows/python-app.yml (hunk 1 above)
Comment (hyunssong) [reply .github/workflows/python-app.yml:66]: I rebased the current branch based on the commit `21829fb30cd94e595b4ff35d7bfeaabcbe3c3e4a` [commit](https://github.com/isamplesorg/isamples_inabox/commit/21829fb30cd94e595b4ff35d7bfeaabcbe3c3e4a) which seemed to be the latest develop when I pushed this commit.  Should I be rebasing based on an older version of `develop`? (the latest version before it got broken)

Comment (dannymandel) [reply .github/workflows/python-app.yml:66]: Given the force push that occurred, you might need to cut a new branch and cherry pick the other commits. This shouldn’t be showing something that has already merged to develop. If this gets merged as is I suspect weird things will happen. You could try doing a merge of latest develop if the rebase doesn’t seem to be working.

Comment (hyunssong) [reply .github/workflows/python-app.yml:66]: Thank you for letting me know. I created a new branch and cherry-picked the commits due to the error in this branch, please see #210.

**Comment by dannymandel** [comment]
Closing this one in favor of https://github.com/isamplesorg/isamples_inabox/pull/210

**Pull request closed without merging**